pages/linux/findfs.md

```diff
@@ -0,0 +1,20 @@
+# findfs
+
+> Finds a filesystem by label or UUID.
+> More information: <https://mirrors.edge.kernel.org/pub/linux/utils/util-linux>.
+
+- Search block devices by filesystem label:
+
+`findfs LABEL={{label}}`
+
+- Search block devices by filesystem UUID:
+
+`findfs UUID={{label}}`
+
+- Search block devices by partition UUID (GPT or MAC partition table):
+
+`findfs PARTLABEL={{label}}`
```

Comment:
maybe we should change "label" here to some that make more sense with the description. `partition_uuid` perhaps

Reply:
also I got this one mixed up with the one below